erpnext/stock/doctype/stock_reconciliation/test_stock_reconciliation.py

```diff
@@ -529,6 +529,7 @@ def test_backdated_stock_reco_cancellation_future_negative_stock(self):
 
 		# check if cancellation of stock reco is blocked
 		self.assertRaises(NegativeStockError, sr.cancel)
+		frappe.db.rollback()
 
 		repost_exists = bool(frappe.db.exists("Repost Item Valuation", {"voucher_no": sr.name}))
```

Comment:
Remove frappe.db.rollback() and add one more condition "status": "Queued" in the filter (534 line) and check whether this test case working or not